### DISCUSSION TITLE
求一个恢复老专栏分类的组件

### DISCUSSION THREAD

**Discussion by RtYkk** [the1812/Bilibili-Evolved#5042] [功能建议 / Ideas]
https://space.bilibili.com/*/article 这个页面现在被傻逼逼站改成新的图文了，动态和专栏混在一起，纯逆天，反倒是早就消失的音频区现在独立了。搞了个所谓的专栏文集还得 up 自己设置
![image](https://github.com/user-attachments/assets/dc3948d5-320f-4225-bb96-c5ce68d1b94f)
我想能不能在图文下面整个新的“专栏”，点击之后在右侧把老专栏搞回来，因为老的 api 目前还有效。自己找 Claude 聊了聊，怎么说怎么做不明白……如果有人能做一个不胜感激，在此提前谢过。
https://github.com/SocialSisterYi/bilibili-API-collect/blob/master/docs/article/list.md

**Comment by Geeyun-JY3** [comment]
请善用搜索，重复：#4598

**Comment by RtYkk** [comment]
sry
